source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl

```diff
@@ -778,19 +778,19 @@ AdaptiveNamespaceStart
 
     [
 #ifdef ADAPTIVE_CARDS_WINDOWS
-        interface_name("Windows.Internal.AdaptiveCards.Rendering.Uwp.IAdaptiveNumberInput", 7c093cbd-cb81-4e3f-b3ae-82664ce87859),
-        contract(InternalContract, 1),
+        interface_name("Windows.Internal.AdaptiveCards.Rendering.Uwp.IAdaptiveNumberInput", c91b6d6d-ee5d-465b-a86e-82f449b7c184),
+        contract(InternalContract, 2),
         internal
 #endif
     ]
     runtimeclass AdaptiveNumberInput : IAdaptiveInputElement, IAdaptiveCardElement
     {
         AdaptiveNumberInput();
 
-        Int32 Max;
-        Int32 Min;
+        Windows.Foundation.IReference<Int32> Max;
```

Comment:
>Windows.Foundation.IReference<Int32> Max; [](start = 8, length = 41)

There's going to need to be some more fixes once this change comes into contact with the validation branch. Might be worth considering going into there even though this isn't technically a validation feature.

Reply:
@paulcam206 might be worth considering basing this against feature/inputEvolution if only to prevent merge issues?

Reply:
> @paulcam206 might be worth considering basing this against feature/inputEvolution if only to prevent merge issues - is there a reason why you decided to go against main directly here?

This functionality isn't dependent on the Input Evolution work, so I just automatically decided to go against `main`. It's no problem to bring the changes over to the IE feature branch, though, so I will do that :)

Reply:
@paulcam206  heads up that we've since merged into main -- so you should be good to continue to go against main at this point!